torch/distributed/_shard/sharded_tensor/_ops/__init__.py

```diff
@@ -2,7 +2,10 @@
 import torch.distributed._shard.sharded_tensor._ops.math_ops
 
 from .binary_cmp import equal, allclose
-from .embedding import sharded_embedding
-from .embedding_bag import sharded_embedding_bag
 from .init import kaiming_uniform_, normal_, uniform_, constant_
-from .linear import sharded_linear
+
+# Import all ChunkShardingSpec ops
+from torch.distributed._shard.sharding_spec.chunk_sharding_spec_ops.linear import sharded_linear
+from torch.distributed._shard.sharding_spec.chunk_sharding_spec_ops.embedding import sharded_embedding
+from torch.distributed._shard.sharding_spec.chunk_sharding_spec_ops.embedding_bag import sharded_embedding_bag
+import torch.distributed._shard.sharding_spec.chunk_sharding_spec_ops.math_ops
```

Comment:
Curious, why not put this into `torch/distributed/_shard/sharding_spec/chunk_sharding_spec_ops/__init__.py` first and we then import that file?